packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCheck.tsx

```diff
@@ -12,31 +12,34 @@ import {
 import { isMobileScreen, getLandscapeMediaQuery } from '../utils/device';
 import { CancelButton } from '../shared/CancelButton';
 import {
+  InstructionDisplayText,
   HintDisplayText,
   CameraDisplayText,
   StreamDisplayText,
   ErrorDisplayText,
   defaultErrorDisplayText,
 } from '../displayText';
 import { LandscapeErrorModal } from '../shared/LandscapeErrorModal';
-import { CheckScreenComponents } from '../shared/FaceLivenessErrorModal';
 import { selectErrorState } from '../shared';
+import { FaceLivenessDetectorComponents } from '../shared/DefaultStartScreenComponents';
 
 const CHECK_CLASS_NAME = 'liveness-detector-check';
 
-const selectIsRecordingStopped = createLivenessSelector(
+export const selectIsRecordingStopped = createLivenessSelector(
   (state) => state.context.isRecordingStopped
 );
 
 interface LivenessCheckProps {
+  instructionDisplayText: Required<InstructionDisplayText>;
   hintDisplayText: Required<HintDisplayText>;
   cameraDisplayText: Required<CameraDisplayText>;
   streamDisplayText: Required<StreamDisplayText>;
   errorDisplayText: Required<ErrorDisplayText>;
-  components?: CheckScreenComponents;
+  components?: FaceLivenessDetectorComponents;
```

Comment:
Prefer scoped `components` interfaces to enforce a separation of concerns between slots